packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneBlobs.spec.ts

```diff
@@ -0,0 +1,192 @@
+    // blob was unreferenced in.
+    const summarizationWithUnreferencedBlobAfterTime =
+    async (approximateUnreferenceTimestampMs: number) => {
+        const container = await makeContainer();
+        const defaultDataObject = await requestFluidObject<ITestDataObject>(container, "default");
+        await waitForContainerConnection(container);
+
+        // Create blob
+        const handleKey = "handle";
+        const blobContents = "Blob contents";
+        const blobHandle = await defaultDataObject._runtime.uploadBlob(stringToBuffer(blobContents, "utf-8"));
+
+        // Reference a blob - important for making it live
+        defaultDataObject._root.set(handleKey, blobHandle);
+
+        // Unreference a blob
+        defaultDataObject._root.delete(handleKey);
+
+        // Summarize
+        const {
+            container: summarizingContainer1,
+            summarizer: summarizer1,
+        } = await loadSummarizerAndContainer();
+        const summaryVersion = (await summarize(summarizer1)).summaryVersion;
+
+        // Close the containers as these containers would be closed by session expiry before sweep ready ever occurs
+        container.close();
+        summarizingContainer1.close();
+
+        // Wait some time, the datastore can be in many different unreference states
+        await delay(approximateUnreferenceTimestampMs);
+
+        // Load a new container and summarizer based on the latest summary, summarize
+        const {
+            container: summarizingContainer2,
+            summarizer: summarizer2,
+        } = await loadSummarizerAndContainer(summaryVersion);
+
+        return {
+            absolutePath: blobHandle.absolutePath,
+            summarizingContainer: summarizingContainer2,
+            summarizer: summarizer2,
+            summaryVersion,
+        };
+    };
+
+    // If this test starts failing due to runtime is closed errors try first adjusting `sweepTimeoutMs` above
+    itExpects("Handle request for tombstoned blobs fails in summarizing container loaded after sweep timeout",
+    [
+        {
+            error: "GC_Tombstone_Blob_Requested",
+            eventName: "fluid:telemetry:BlobManager:GC_Tombstone_Blob_Requested",
+            viaHandle: true,
+        },
+    ],
+    async () => {
+        const {
+            absolutePath,
+            summarizingContainer,
+            summarizer,
+        } = await summarizationWithUnreferencedBlobAfterTime(sweepTimeoutMs);
+        // The blob should be tombstoned now
+        const { summaryVersion } = await summarize(summarizer);
+
+        const container = await loadContainer(summaryVersion);
+        const defaultDataObject = await requestFluidObject<ITestDataObject>(container, "default");
+        const fluidHandleContext = defaultDataObject._context.containerRuntime.IFluidHandleContext;
+
+        // Handle requests for blob handle should fail!
+        const response = await fluidHandleContext.resolveHandle({ url: absolutePath });
```

Comment:
You should get the blob via its handle not via request / resolveHandle because that is what will happen in real scenarios.
This is fine to keep too but getting from handle is important as well.

Reply:
I'm going to have to create a handle or something because there's no way of getting the handle from the container as the blob doesn't exist.

Reply:
Oh I see. I didn't look into detail and assumed that it's the same container.